kafka-connect/kafka-connect-runtime/src/integration/java/org/apache/iceberg/connect/KafkaConnectUtils.java

```diff
@@ -54,10 +54,6 @@ public String getName() {
       return name;
     }
 
-    public Map<String, Object> getConfig() {
```

Comment:
Why are we skipping the deprecation process?

Reply:
I did not see `KafkaConnectUtils` in the [Javadoc](https://iceberg.apache.org/javadoc/nightly/), so I assumed it was not part of the public API and did not need a deprecation. 

Anyway this change is the reason the tests are broken, it turns out this method is required.

I've reverted this change.